sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Mean.java

```diff
@@ -156,6 +156,10 @@ public Double extractOutput() {
     }
 
     @Override
+    @edu.umd.cs.findbugs.annotations.SuppressWarnings(
```

Comment:
Neat! I didn't know that this worked. I know that at HEAD the findbugs annotations have `@SuppressFBWarning` but if we don't need it that is good since the project looks stale.